src/stores/ContractsStore.ts

```diff
@@ -1,48 +1,44 @@
 import { ContractsStore } from '@big-whale-labs/stores'
-import {
-  goerliDefaultProvider,
-  mainnetDefaultProvider,
-} from 'helpers/providers/defaultProvider'
-import {
-  goerliHeavyProvider,
-  mainnetHeavyProvider,
-} from 'helpers/providers/heavyProvider'
 import { proxy } from 'valtio'
 import { subscribeKey } from 'valtio/utils'
 import Network from 'models/Network'
 import WalletStore from 'stores/WalletStore'
 import env from 'helpers/env'
+import networks from 'networks'
+import transformObjectValues from 'helpers/transformObjectValues'
 
-export const GoerliContractsStore = proxy(
-  new ContractsStore(goerliDefaultProvider, goerliHeavyProvider, Network.Goerli)
-).makePersistent(env.VITE_ENCRYPT_KEY)
+const proxyNetworks = transformObjectValues(
+  networks,
+  ({ network, heavyProvider, defaultProvider }) =>
+    proxy(
+      new ContractsStore(defaultProvider, heavyProvider, network)
+    ).makePersistent(env.VITE_ENCRYPT_KEY)
+)
 
-export const MainnetContractsStore = proxy(
-  new ContractsStore(
-    mainnetDefaultProvider,
-    mainnetHeavyProvider,
-    Network.Mainnet
-  )
-).makePersistent(env.VITE_ENCRYPT_KEY)
+const ContractsNetworkStore = proxy({
+  networks: proxyNetworks,
+})
+
+export const BadgesNetwork = Network.Goerli
```

Comment:
Do we need this? We know, that by default we're using `goerli` 🤔

Reply:
And if something changes, we will definitely change everything right?